experiment.py

```diff
@@ -39,7 +43,24 @@ def __enter__(self):
 
     def __exit__(self, typ=None, value=None, traceback=None):
         self.xp_state.eg.serialize()
-        ag.commit(self.xp_state)
+        original = os.getcwd()
+        if os.path.exists(self.xp_state.versioningDirectory + '/' + self.xp_state.EXPERIMENT_NAME):
+            move(self.xp_state.versioningDirectory + '/' + self.xp_state.EXPERIMENT_NAME + '/.git', '/tmp/')
+            rmtree(self.xp_state.versioningDirectory + '/' + self.xp_state.EXPERIMENT_NAME)
+            move('/tmp/.git', self.xp_state.versioningDirectory + '/' + self.xp_state.EXPERIMENT_NAME + '/.git')
+            copytree(os.getcwd(), self.xp_state.versioningDirectory + '/' + self.xp_state.EXPERIMENT_NAME + '/0')
```

Comment:
Note to self: if running flor plan from other directory, I think I have feature implemented so that flor automatically changes directory to the right one. need to check...